charts/posit-chronicle/Chart.yaml

```diff
@@ -1,27 +1,37 @@
 apiVersion: v2
 name: posit-chronicle
 description: Official Helm chart for Posit Chronicle Server
-version: 0.3.8
+version: 0.4.0
 appVersion: 2025.03.0
-icon: https://rstudio.com/wp-content/uploads/2018/10/RStudio-Logo-Flat.png
+icon: https://posit.co/wp-content/themes/Posit/dist/images/favicon/apple-touch-icon-180x180.png
 home: https://www.posit.co
 sources:
   - https://github.com/rstudio/helm
 maintainers:
-  - name: sol-eng
+  - name: rstudio
     email: docker@posit.co
     url: https://github.com/rstudio/helm
 annotations:
+  artifacthub.io/images: |
+    - name: chronicle
+      image: ghcr.io/rstudio/chronicle:2025.03.0
```

Comment:
Just a question here - is the image name override-able? I'm thinking about future cases where we want to deploy a RC build to posit.it before we release it publicly. 

I see https://github.com/rstudio/helm/blob/main/charts/posit-chronicle/values.yaml#L2, so I _think_ a user could point to a different container registry (e.g., use ECR instead of GHCR).